src/ray/core_worker/core_worker.cc

```diff
@@ -492,12 +491,17 @@ Status CoreWorker::Contains(const ObjectID &object_id, bool *has_object) {
     // ErrorType::OBJECT_IN_PLASMA.
     found = memory_store_->Contains(object_id);
   }
+  /*
+  // We don't need this if we hold the invariant that all the direct call
```

Comment:
Not sure I understand this. According to the comment above, the memory store will return `false` if the object value is `OBJECT_IN_PLASMA`, so we still need to check, right?.